.github/workflows/post.yml

```diff
@@ -128,13 +133,19 @@ jobs:
             quay.io/open-cluster-management/addon-manager:latest-amd64 --arch amd64
           docker manifest annotate quay.io/open-cluster-management/addon-manager:latest \
             quay.io/open-cluster-management/addon-manager:latest-arm64 --arch arm64
+          # aws-cli
+          docker manifest annotate quay.io/open-cluster-management/aws-cli:latest \
+            quay.io/open-cluster-management/aws-cli:latest-amd64 --arch amd64
+          docker manifest annotate quay.io/open-cluster-management/aws-cli:latest \
+            quay.io/open-cluster-management/aws-cli:latest-arm64 --arch arm64
       - name: push
         run: |
           docker manifest push quay.io/open-cluster-management/registration-operator:latest
           docker manifest push quay.io/open-cluster-management/registration:latest
           docker manifest push quay.io/open-cluster-management/work:latest
           docker manifest push quay.io/open-cluster-management/placement:latest
           docker manifest push quay.io/open-cluster-management/addon-manager:latest
+          docker manifest push quay.io/open-cluster-management/aws-cli:latest
```

Comment:
Thanks for updating this file. We will need similar changes in ``.github/workflows/releaseimage.yml` as well.